src/Admin/OrderAdminExtension.php

```diff
@@ -42,16 +42,15 @@ public function addOrderMetaBoxActions($actions)
         $order = $this->getCurrentOrder();
 
         if ($order) {
-            $status = $order->get_meta(OrderTransmitter::STATUS_META_KEY);
-
-            if (!$status || substr($status, 0, 5) === 'ERROR') {
+            $orderTransmitter = OrderTransmitterFactory::create();
+            if (!$orderTransmitter->hasAlreadyBeenTransmitted($order)) {
                 $actions['penny_black_send'] = 'Send to Penny Black';
-            } else {
-                $orderAdminEnabled = \WC_Admin_Settings::get_option(Settings::FIELD_ENABLE_ORDER_EXTENSIONS);
+            }
+
+            $orderAdminEnabled = \WC_Admin_Settings::get_option(Settings::FIELD_ENABLE_ORDER_EXTENSIONS);
 
-                if ($orderAdminEnabled && $orderAdminEnabled !== 'no') {
-                    $actions['penny_black_print'] = 'Print via Penny Black';
-                }
+            if ($orderAdminEnabled && $orderAdminEnabled !== 'no') {
+                $actions['penny_black_print'] = 'Print via Penny Black';
```

Comment:
it's possible that you could get an error of "already sent" for an order (which you can't recover from) so I removed the else on this condition to ensure we always have a print option visible